lib/convict.js

```diff
@@ -401,7 +407,7 @@ var convict = function convict(def) {
       var self = this;
       if (!Array.isArray(paths)) paths = [paths];
       paths.forEach(function(path) {
-        overlay(cjson.load(path), self._instance, self._schema);
+        overlay(loadJSON(path), self._instance, self._schema);
```

Comment:
cjson uses sync everywhere: https://github.com/kof/node-cjson/blob/master/index.js#L221. I think it may be tricky to suddenly switch to async.